umm-spec-lib/test/cmr/umm_spec/test/migration/version/granule.clj

```diff
@@ -202,3 +203,189 @@
+                         :Type "GET DATA"
+                         :Subtype "USER FEEDBACK PAGE"}
+                        {:URL "https://example-2.hdf"
+                         :Type "GET DATA"
+                         :Subtype "OPENDAP DATA"
+                         :MimeType "application/x-hdf5"}]}
+        (vm/migrate-umm {} :granule "1.6" "1.5"
+                        {:MetadataSpecification
+                           {:URL "https://cdn.earthdata.nasa.gov/umm/granule/v1.5"
+                            :Name "UMM-G"
+                            :Version "1.5"}
+                         :DataGranule {:ArchiveAndDistributionInformation
+                                         [{:Name "GranuleZipFile"
+                                           :Size 23
+                                           :SizeUnit "KB"
+                                           :Format "ZIP"
+                                           :MimeType "application/x-hdf5"
+                                           :Files [{:Name "GranuleFileName1"
+                                                    :Size 10
+                                                    :SizeUnit "KB"
+                                                    :Format "NETCDF-4"
+                                                    :MimeType "application/x-netcdf"
+                                                    :FormatType "Native"
+                                                    :SizeInBytes 1024}
+                                                   {:Name "GranuleFileName2"
+                                                    :Size 1
+                                                    :SizeUnit "KB"
+                                                    :Format "ASCII"
+                                                    :MimeType "application/x-hdf5"
+                                                    :FormatType "NA"
+                                                    :SizeInBytes 10248592}]}
+                                          {:Name "SupportedGranuleFileNotInPackage"
+                                           :Size 11
+                                           :SizeUnit "KB"
+                                           :SizeInBytes 849324895784093
+                                           :Format "NETCDF-CF"
+                                           :FormatType "Supported"
+                                           :MimeType "application/x-netcdf"}]
+                                        :Identifiers
+                                         [{:Identifier "acos_LtCO2_090421_v201201_B7310A_161220013536s.nc4 and here are some extra characters so that this is too long for 1.5"
+                                           :IdentifierType "ProducerGranuleId"
+                                           :IdentifierName "Now this is the story all about how my life got flip-turned upside-down and I'd like to take a minute just sit right there"}]}
```

Comment:
Can you add a test case where the `IdentifierName` does not exist in v1.6? I don't think your code handles this case correctly.

Reply:
Fixed